MightyMike.spec

```diff
@@ -0,0 +1,69 @@
+#
+# spec file for package MightyMike by <https://www.notabug.org/tux_peng>
+#
+# MightyMike is a registered trademark of Pangea Software, Inc.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://github.com/jorio/MightyMike/issues
+#
+%global debug_package %{nil}
+
+
+Name:           MightyMike
+Version:        3.0.1
+Release:        0
+Summary:        It's up to MightyMike to save the innocent fuzzy bunnies
+License:        CC-BY-NC-SA-4.0
+Group:          Amusements/Games/3D/Other
+URL:            https://github.com/jorio/MightyMike/
+Source:         MightyMike-%{version}.tar.gz
```

Comment:
Is it possible to have OBS pull the source directly from github rather than shipping a tarball?